plugins/modules/zos_backup_restore.py

```diff
@@ -1005,12 +991,16 @@ def to_dunzip_args(**kwargs):
             size += kwargs.get("space_type")
         zoau_args["size"] = size
 
-    if kwargs.get("hlq"):
-        zoau_args["high_level_qualifier"] = kwargs.get("hlq")
-
     if kwargs.get("tmp_hlq"):
         zoau_args["tmphlq"] = str(kwargs.get("tmp_hlq"))
 
+    if kwargs.get("hlq") is None:
+        zoau_args["high_level_qualifier"] = datasets.get_hlq()
+    elif kwargs.get("hlq") == "K" or kwargs.get("hlq") == "R":
```

Comment:
im a bit unclear here, what is the point of introducing two new keywords if they both do the same thing?